conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+bot:
+  abi_migration_branches: [v18]
```

Comment:
@bgruening this will ensure that the bot keeps sending migration PRs to this branch.

Reply:
Oh that is cool! Thanks!